subsys/bluetooth/controller/hci_driver.c

```diff
@@ -490,6 +490,14 @@ static int configure_supported_features(void)
 		if (err) {
 			return -ENOTSUP;
 		}
+
+		sdc_hci_cmd_vs_periodic_adv_event_length_set_t params = {
+			.event_length_us = CONFIG_BT_CTLR_SDC_PERIODIC_ADV_EVENT_LEN_DEFAULT
+		};
+		err = sdc_hci_cmd_vs_periodic_adv_event_length_set(&params);
```

Comment:
I'm not sure it's good to call HCI commands before sdc_enable. Can we move this call to hci_driver_open instead?